tenets/codelingo/effective-go/comment-first-word-as-subject/codelingo.yaml

```diff
@@ -157,7 +157,8 @@ tenets:
           The first sentence should be a one-sentence summary that starts with the name being declared.
       codelingo/review:
         comment: |
-          The first sentence should be a one-sentence summary that starts with the name ({{funcName}}) being declared.
+          Every exported function in a program should have a doc comment. The first sentence should be a one-sentence summary that starts with the name ({{funcName}}) being declared.
```

Comment:
The first sentence should be a summary that starts with .... haha obviously the first sentence will be one sentence